### DISCUSSION TITLE
become moderator

### DISCUSSION THREAD

**Discussion by GamingFloppaYT** [maxnut/GDmenu#276] [Q&A]
i cant find become moderator button

**Comment by XenonPhanes** [comment]
that's a separate mod in geode

**Reply by GamingFloppaYT** [reply]
no i remember there was a button called become moderator

**or maybe i dreamed it...** but if it was a dream everything was normal so it couldnt have been a dream, i remember it was in mega overlay or else megahack

**Reply by saverofthewos** [reply]
i think theres a bunch of stuff missing in the 2.2/geode version
